pkg/manager/manager.go

```diff
@@ -142,11 +142,15 @@ type Options struct {
 	// starting the manager.
 	LeaderElection bool
 
+	// LeaderElectionResourceLock determines which resource lock to use for leader election,
+	// defaults to "configmapsleases".
+	LeaderElectionResourceLock string
```

Comment:
Sorry, I forgot to press submit on my comment yesterday. Please mention here that ppl should only change this if they know what they are doing and that they must make sure all their users are transitiion to the new lock via a multilock to the new lock before they change this, keeping in mind that users might skip versions of the controller.

Reply:
Done.